### DISCUSSION TITLE
End Entity Profile Limitations

### DISCUSSION THREAD

**Discussion by Shree1-debug** [Keyfactor/ejbca-ce#619] [Q&A]
Can we enable and disable "End Entity Profile Limitations" which is in the System Configuration, using command line interface as we need to automate this. Please provide the command, as we have already tried the following ways but it did not work :

- ./ejbca.sh config setproperty org.ejbca.core.ejb.ra.raadmin.EndEntityProfileUse=false
- org.ejbca.ui.web.protocol.alloweeprofilelimitations=false

we directly tried to add/set these properties in the ejbca.properties file and restarted the ejbca container, but still it is not disabling the end entity profile limitations.

Please let us know if any way is present to disable this from backed. 

For your reference, please find the attached screenshot which is highlighted.
![image](https://github.com/Keyfactor/ejbca-ce/assets/174118673/60af9fc6-3ce0-4cd1-b24e-66ed27248dea)

**Comment by primetomas** [comment]
This is in Global Configuration, (GlobalConfigurationData in the database). This setting can not be automated at the moment. 
Enabling/disabling this field affect global operations, across all protocols and profiles, so it's not really advised to modify this like that.

Why do you think you need to disable this field? It's typically better to create profile that are open ended enough to allow the fields you want to add to end entities. Or use the "Override" settings in certificate profiles.
